src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.jsx

```diff
@@ -1,8 +1,8 @@
-import React, { useRef, useCallback, useEffect } from "react";
+import React, { useRef, useCallback, useEffect, useMemo } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
 import NOOP from "lodash/noop";
-import { COLOR_STYLES } from "../../../../general-stories/colors/colors-vars-map";
+import { COLOR_STYLES, contentColors } from "../../../../general-stories/colors/colors-vars-map";
```

Comment:
can we move these to something not related to stories ?

Reply:
I'm not sure I understand you correctly - right now it's not related to storybook. what are you suggesting?

Reply:
I think it's about to move this constants to different location which not related to stories, maybe the colors util?

Reply:
oh I see it now! done